app/policies/application_policy.rb

```diff
@@ -9,31 +9,31 @@ def initialize(user, record)
   end
 
   def index?
-    false
+    user.admin?
```

Comment:
I tend to not include this at the application_policy level because there are times where you don't want even admin to have access (and I much prefer being explicit per policy file). You can 100% alias inside the policy file though.